pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -2253,14 +2253,15 @@ var _ = Describe("Manager helper functions", func() {
 
 			properDisk = api.Disk{
 				FilesystemOverhead: &fakePercent,
-				Capacity:           resource.NewScaledQuantity(123, 4),
+				StorageCapacity:    resource.NewScaledQuantity(123, 4),
+				StorageRequest:     resource.NewScaledQuantity(125, 4),
```

Comment:
can we have a unit test that ensures that the minimum of these two values is selected. This would give us some guarantee that this behavior will not change in the future.

Reply:
Added a unit test in converter_test.go for this.